Site/ASAB Maestro/Files/nginx/conf.d/server_https_default.conf

```diff
@@ -0,0 +1,16 @@
+server {
+	http2 on;
+	
+	# Match anything that's NOT the main public domain
+	server_name _;
```

Comment:
move it to nginx.yaml descriptor so that both the servers have the same directives on the same place